### PR TITLE
new: Acess File With Common Registry Extention

### PR DESCRIPTION
**Pull request by frack113**
### Summary of the Pull Request

REF : https://github.com/tccontre/Reg-Restore-Persistence-Mole

Detect when an application read registry file (commun extension)
Registry file are textfile.

### Example Log Event

```xml
- <EventData>
  <Data>Sigma rule match found: Create File With Commun Registry Extention (see Details tab for more information)</Data> 
  <Data>Module: Sigma</Data> 
  <Data>Rule_Title: Create File With Commun Registry Extention</Data> 
  <Data>Rule_Author: frack113</Data> 
  <Data>Rule_Description: Detects the creation of a commun registry export file extention. Adversaries may interact with the Windows Registry to gather information about the system, configuration, and installed software.</Data> 
  <Data>Rule_FalsePositives: Legitimate software may create these files (for any reason).</Data> 
  <Data>Rule_Id: 337a31c6-46c4-46be-886a-260d7aa78cac</Data> 
  <Data>Rule_Level: medium</Data> 
  <Data>Rule_Modified: 2023/08/26</Data> 
  <Data>Rule_Path: sigma-rules\file_access_win_suspicius_registry.yml</Data> 
  <Data>Rule_References: https://github.com/tccontre/Reg-Restore-Persistence-Mole</Data> 
  <Data>Rule_Sigtype: custom</Data> 
  <Data>CommandLine: RegReeper.exe</Data> 
  <Data>Computer: Win11</Data> 
  <Data>Correlation_ActivityID: {00000000-0000-0000-0000-000000000000}</Data> 
  <Data>CreateAttributes: 0x0</Data> 
  <Data>CreateOptions: 0x1200044</Data> 
  <Data>EventID: 12</Data> 
  <Data>Execution_ThreadID: 14476</Data> 
  <Data>FileName: \Device\HarddiskVolume3\temp\RegReeper\save_reg.hive</Data> 
  <Data>FileObject: 0xFFFFE506BEA12AC0</Data> 
  <Data>Hashes: MD5=2CCD050BAFA8772966D334EEF4D4A9CA,SHA1=4B626394DBFFF9FCFF960C5AB94D71C295CCC095,SHA256=10C8867FC29CD7CEE45E5035392D060DB8C0671FBE051667981824B8116E9AB7,IMPHASH=BD358AF0967963B052CFAF3A3D91AE28</Data> 
  <Data>Image: C:\temp\RegReeper\RegReeper.exe</Data> 
  <Data>Irp: 0xFFFFE506BBBA20F8</Data> 
  <Data>IssuingThreadId: 14476</Data> 
  <Data>Keywords: 0x80000000000000a0</Data> 
  <Data>Level: 4</Data> 
  <Data>Match_Strings: .hive in FileName</Data> 
  <Data>Opcode: 0</Data> 
  <Data>ParentCommandLine: "C:\WINDOWS\system32\cmd.exe"</Data> 
  <Data>ParentImage: C:\Windows\System32\cmd.exe</Data> 
  <Data>ProcessId: 14316</Data> 
  <Data>Provider_Guid: {EDD08927-9CC4-4E65-B970-C2560FB5C289}</Data> 
  <Data>Provider_Name: Microsoft-Windows-Kernel-File</Data> 
  <Data>Security_UserID: S-1-5-21-888117185-644776935-3477416708-1104</Data> 
  <Data>ShareAccess: 0x5</Data> 
  <Data>TargetFilename: C:\temp\RegReeper\save_reg.hive</Data> 
  <Data>Task: 12</Data> 
  <Data>TimeCreated_SystemTime: 2023-08-26T12:06:22.7695751+02:00</Data> 
  <Data>User: LAB\admin</Data> 
  <Data>Version: 1</Data> 
  <Data>Winversion: 22621</Data> 
  </EventData>
```
### SigmaHQ Rule Creation Conventions

- If your PR adds new rules, please consider following and applying these [conventions](https://github.com/SigmaHQ/sigma-specification/blob/main/sigmahq/sigmahq_conventions.md)

### CODE REVIEW

File: rules/windows/file/file_access/file_access_win_suspicius_registry.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
+title: Access File With Commun Registry Extension
```
Comment (phantinuss) [inline rules/windows/file/file_access/file_access_win_suspicius_registry.yml:1]:
```suggestion
title: Access File With Common Registry Extension
```

**Comment by phantinuss** [comment]
Is this rule about file access or file create?

**Comment by frack113** [comment]
The file is for file access ( `category: file_access`)
I don't see the need for a registry backup rule (`category: file_event`).

File: rules/windows/file/file_access/file_access_win_suspicius_registry.yml [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+            - '.reg'
+    filter_system_folders:
+        Image|startswith:
+            - 'C:\Program Files\'
+            - 'C:\Program Files (x86)\'
+            - 'C:\Windows\system32\'
+            - 'C:\Windows\SysWOW64\'
+    condition: selection and not 1 of filter_*
+falsepositives:
+    - Legitimate software may create these files (for any reason).
```
Comment (phantinuss) [inline rules/windows/file/file_access/file_access_win_suspicius_registry.yml:31]:
```suggestion
    - Legitimate software accessing these files (for any reason).
```

File: rules/windows/file/file_access/file_access_win_suspicius_registry.yml [hunk 3]
```diff
@@ -0,0 +1,32 @@
+title: Access File With Common Registry Extension
+id: 337a31c6-46c4-46be-886a-260d7aa78cac
+status: experimental
+description: |
+  Detects access to a registry export file with a common extension.
+  Adversaries may interact with the Windows Registry to gather information about the system, configuration, and installed software.
```
Comment (phantinuss) [inline rules/windows/file/file_access/file_access_win_suspicius_registry.yml:6]:
```suggestion
  Detects access to exported  Windows Registry files with a .hive or .reg extension
  Adversaries may interact with the Windows Registry to gather information about the system, configuration or installed software.
```

**Review by phantinuss — APPROVED** [review]

File: rules/windows/file/file_access/file_access_win_suspicius_registry.yml [hunk 4]
```diff
@@ -0,0 +1,32 @@
+title: Access File With Common Registry Extension
```
Comment (phantinuss) [inline rules/windows/file/file_access/file_access_win_suspicius_registry.yml:1]:
```suggestion
title: Access to Registry File Exports
```

File: rules/windows/file/file_access/file_access_win_suspicius_registry.yml [hunk 5]
```diff
@@ -0,0 +1,32 @@
...
+id: 337a31c6-46c4-46be-886a-260d7aa78cac
+status: experimental
+description: |
+  Detects access to exported  Windows Registry files with a .hive or .reg extension
```
Comment (phantinuss) [inline rules/windows/file/file_access/file_access_win_suspicius_registry.yml:5]:
```suggestion
  Detects access to exported  Windows Registry files with a .hive or .reg extension.
```

**Review by nasbench — APPROVED** [review]

**Pull request merged**